plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/PluginInputFormatException.java

```diff
@@ -0,0 +1,12 @@
+package src.main.java.io.jenkins.tools.pluginmanager.cli;
```

Comment:
Package is wrong here
```suggestion
package io.jenkins.tools.pluginmanager.cli;
```